packages/plugins/color-picker/admin/src/index.ts

```diff
@@ -1,11 +1,11 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
 
 import ColorPickerIcon from './components/ColorPicker/ColorPickerIcon';
-import pluginId from './pluginId';
+import { pluginId } from './pluginId';
 import getTrad from './utils/getTrad';
 
 export default {
-  register(app) {
+  register(app: any) {
```

Comment:
more for my own understanding, is `app` – `import { Strapi } from '@strapi/strapi'`?

Reply:
yes app will be Strapi but from admin in that case as it's the FE. 

Longer term I would like us to just do that

```typescript
export default definePlugin({
  register(app) {
  
  }
})

// or 
import type { Plugin } from '@strapi/....' 

const plugin: Plugin = {
 register(app) {
 
 }
}

export default plugin
```